app/components/dashboard/card/tooltip_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<span class="p-relative t--2px">
+  <i
+    class="fas fa-question-circle fs-14px c-grey-dark"
+    data-toggle="tooltip"
+    data-placement="right"
+    data-trigger="hover click"
+    data-html="true"
+    <% if definitions.is_a? String %>
+      title="<%= definitions %>"
+    <% else %>
+      title="<% definitions.each do |name, description| %>
+               <p class='mb-4px'><strong><%= name %>:</strong> <%= description %></p>
+             <% end %>"
```

Comment:
This logic can belong to a `render_definitions` method in the component. 

What's the case where we want to pass in a string? Wasn't able to find one in this PR.

Reply:
> What's the case where we want to pass in a string? Wasn't able to find one in this PR.
When we need to render just a summary instead of a key-value pair type definition.
Like the tooltips in the `hypertension/diabetes patients` card under the details section